pkg/kubelet/volumemanager/populator/desired_state_of_world_populator_test.go

```diff
@@ -85,6 +85,71 @@ func prepareDswpWithVolume(t *testing.T) (*desiredStateOfWorldPopulator, kubepod
 	return dswp, fakePodManager
 }
 
+func TestFindAndAddNewPods_WithRescontructedVolume(t *testing.T) {
+	// create dswp
+	dswp, fakePodManager := prepareDswpWithVolume(t)
+
+	// create pod
+	containers := []v1.Container{
+		{
+			VolumeMounts: []v1.VolumeMount{
+				{
+					Name:      "dswp-test-volume-name",
+					MountPath: "/mnt",
+				},
+			},
+		},
+	}
+	pod := createPodWithVolume("dswp-test-pod", "dswp-test-volume-name", "file-bound", containers)
```

Comment:
how about make "dswp-test-volume-name" as a variable (outername) to make it easy to check